### DISCUSSION TITLE
Error response from daemon: Get "https://ghcr.io/bla": unauthorized

### DISCUSSION THREAD

**Discussion by Zethson** [community/community#26015] [Actions]
<p>Dear everyone,</p>
<p>I published a docker image as a github package:<br>
Repository: <a href="https://github.com/Zethson/exploding_springfield" rel="nofollow noopener">https://github.com/Zethson/exploding_springfield</a><br>
using the workflow:<br>
<a href="https://github.com/Zethson/exploding_springfield/actions/runs/239922181" rel="nofollow noopener">https://github.com/Zethson/exploding_springfield/actions/runs/239922181</a></p>
<p>The package build successfully and I can view it.<br>
But ONLY when I am logged in to Github? Why?</p>
<p>It is a public repository. Why is the package not public?</p>
<p>Thanks.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>:wave: I can see from your Action run that it looks like you published <a href="http://ghcr.io/zethson/exploding_springfield/exploding_springfield">ghcr.io/zethson/exploding_springfield/exploding_springfield</a> with tags <code>0.1.0-SNAPSHOT</code> and <code>latest</code> which should mean there’s a container now living at <a href="https://github.com/users/zethson/packages/container/exploding_springfield/exploding_springfield">https://github.com/users/zethson/packages/container/exploding_springfield/exploding_springfield</a></p>
<p>Containers all start out out private and need to be switched to public via their settings.</p>

**Comment by lakminagamage** [comment]
if you get "Error response from daemon: Head "https://ghcr.io/bla": unauthorized" in amazon lightsail, you are probably using docker using sudo. Due to a reason that i don't understand, it doesn't work with sudo.
As a workaround for this add docker user to the Docker group.
https://www.geeksforgeeks.org/how-to-add-user-to-docker-group/
